pandas/tests/io/formats/style/test_to_latex.py

```diff
@@ -242,6 +242,25 @@ def test_multiindex_row(df):
     assert expected == s.to_latex(sparse_index=False)
 
 
+def test_multirow_naive(df):
+    ridx = MultiIndex.from_tuples([("A", "a"), ("A", "b"), ("B", "c")])
+    df.loc[2, :] = [2, -2.22, "de"]
+    df = df.astype({"A": int})
+    df.index = ridx
+    expected = dedent(
+        """\
+        \\begin{tabular}{llrrl}
+         &  & A & B & C \\\\
+        A & a & 0 & -0.61 & ab \\\\
+         & b & 1 & -1.22 & cd \\\\
+        B & c & 2 & -2.22 & de \\\\
+        \\end{tabular}
+        """
+    )
+    s = df.style.format(precision=2)
+    assert expected == s.to_latex(multirow_align="naive")
```

Comment:
I suggest that ``result`` variable is extracted:
``` python
result = s.to_latex(multirow_align="naive")
assert result == expected
```

Besides, ``s`` is not a preferred variable name, because one-letter variables make debugging more complicated.
What about making it more explicit, ``styler``?

Reply:
yes cleaned up module wide (somewhat)